nengo/utils/compat.py

```diff
@@ -30,6 +32,13 @@ def ensure_bytes(s):
         assert isinstance(s, bytes)
         return s
 
+    def replace(src, dst):
```

Comment:
Yeah, looking at the `shutil.move` source, it actually does a copy in the case of OSError on rename, and then deletes the source. The copy function they use will actually overwrite I think, but it seems like some extra work to copy the file. So I'd probably just go with this, unless there's a clear way in which `shutil.move` is more robust.